src/include/execution/compiler/operator/operator_translator.h

```diff
@@ -122,6 +125,22 @@ class OperatorTranslator : public ExpressionEvaluator {
    */
   virtual ast::Expr *GetOutput(uint32_t attr_idx) = 0;
 
+  /**
+   * @return The list of extra fields added to the main work function. This is only called on
+   *         parallel pipelines and only for the source/root of the pipeline that's responsible for
+   *         launching the worker function. By default, the first two argument of a parallel work
+   *         function are the query state and thread-local pipeline state.
+   */
+  virtual util::RegionVector<ast::FieldDecl *> GetWorkerParams() = 0;
```

Comment:
yeah like here it's default is `UNREACHABLE`?

Reply:
Right this method should only be used by parallel pipelines so it's default to be unreachable for others